index.html

```diff
@@ -0,0 +1,146 @@
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- Main is divided into three sections with the img and description   -->
+
+    <div class="hero"></div>
+
+    <!-- Added a id "search-engine-optimization" to fix the nav link -->
+
+    <main class="content">
+      <section>
+        <div id="search-engine-optimization" class="search-engine-optimization">
+          <img
+            src="./assets/images/search-engine-optimization.jpg"
+            alt="Alternative options for business"
+            class="float-left"
+          />
+
+          <h2>Search Engine Optimization</h2>
+          <p>
+            The dominance of mobile internet use means that users are searching
+            for the right business as they travel, shop, or sit on their couch
+            at home. Search Engine Optimization (SEO) allows you to increase
+            your visibility and find the right customers for your business.
+          </p>
+        </div>
+      </section>
+
+      <section
+        id="online-reputation-management"
+        class="online-reputation-management"
+      >
+        <img
+          src="./assets/images/online-reputation-management.jpg"
+          alt="Control over digital footprint "
+          class="float-right"
+        />
+
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </section>
+
+      <section id="social-media-marketing" class="social-media-marketing">
+        <img
+          src="./assets/images/social-media-marketing.jpg"
+          alt="customers in right media"
+          class="float-left"
+        />
+
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </section>
+    </main>
+
+    <!--Aside tags were assigned for tree parts in this section -->
+    <aside class="benefits">
+      <section class="benefit">
+        <h3>Lead Generation</h3>
+        <img
+          src="./assets/images/lead-generation.png"
+          alt="Identify your customers"
+        />
+
```

Comment:
Try to keep spacing consistent. It's great between sections to separate them out, but try to avoid random spacing like this one